bindings/python/iota_sdk/client/client.py

```diff
@@ -414,7 +414,7 @@ def generate_addresses(self,
 
```

Comment:
Can't comment there, but I guess the function should also be renamed to `generate_ed25519_addresses`

Reply:
I left the client alone everywhere else (I think).

Reply:
Fine for me, we should maybe remove it from the client alltogether at some point